src/source/CppMarshal.scala

```diff
@@ -6,27 +6,38 @@ import djinni.meta._
 
 class CppMarshal(spec: Spec) extends Marshal(spec) {
 
```

Comment:
The reason why I didn't include "conflicting" in here is because the set of names that is passed in is (ideally) the whole set of names, and typically most of them are not in conflict. So I guess the correct term would be `potentiallyConflictingSymbolsInScope`, but that's pretty long. Making the variable name much longer also makes it necessary to expand a good number of function definitions from a single line to three, which I thought makes the code less readable as well.

I don't have a good answer to all of these concerns, but if less compact code is not so much of an issue, I think potentiallyConflictingSymbolsInScope is not the worst choice, it's expressive enough to convey at least some of its purpose immediately. I'll add a comment somewhere around the top of the CppMarshal class.